src/views/dropdown.js

```diff
@@ -60,6 +62,21 @@ export default Marionette.LayoutView.extend({
       _.defer(this.showList.bind(this))
     }
   },
+  onButtonFocus: function (e) {
+    if (!this.expanded) {
+      _.defer(this.showList.bind(this))
```

Comment:
Can't really remember now. My guess is that I did it that wat simply because that's the way that it was done in onButtonClick.